client/mysqltest.cc

```diff
@@ -6016,9 +6016,10 @@ void do_connect(struct st_command *command)
   {
     /*
       If the socket is specified just as a name without path
+      or an abstract socket indicator ('@'), then
       append tmpdir in front
     */
-    if (*ds_sock.str != FN_LIBCHAR)
+    if (*ds_sock.str != FN_LIBCHAR && *ds_sock.str != '@')
```

Comment:
No action needed, but did you check the other clients?

Reply:
yes, the rest pass the socket straight though to the connector-c api.